src/main/java/eu/europa/ted/efx/interfaces/TranslatorDependencyFactory.java

```diff
@@ -52,7 +52,7 @@ public interface TranslatorDependencyFactory {
    *        language features that ScriptGenerator instance should be able to handle.
    * @return An instance of ScriptGenerator to be used by the EFX translator.
    */
-  public ScriptGenerator createScriptGenerator(String sdkVersion);
+  public ScriptGenerator createScriptGenerator(String sdkVersion, TranslatorOptions options);
```

Comment:
This is breaking for third-party implementation of this interface (introduced in PR #68 ).
One option to avoid this could be to instead add a new method with a default implementation and deprecate the existing one:

```
   * @deprecated use {@link #createScriptGenerator(String, TranslatorOptions)} instead. 
   */
  @Deprecated(since = "1.3.0", forRemoval = true)
  public ScriptGenerator createScriptGenerator(String sdkVersion);

  default public ScriptGenerator createScriptGenerator(String sdkVersion, TranslatorOptions options) {
    return createScriptGenerator(sdkVersion);
  }
```